technitium-dns/umbrel-app.yml

```
@@ -0,0 +1,25 @@
+manifestVersion: 1
+id: technitium-dns
+category: networking
+name: Technitium DNS server
+version: "12.1"
+tagline: Block ads & malware at DNS level for your entire network!
+description: >-
+  Technitium DNS Server is an open source authoritative as well as recursive DNS server that can be used for self hosting a DNS server for privacy & security. It works out-of-the-box with no or minimal configuration and provides a user friendly web console accessible using any modern web browser.
+
+developer: Shreyas Zare
+website: https://technitium.com/dns/
+dependencies:
+repo: https://github.com/TechnitiumSoftware/DnsServer
+support: https://github.com/TechnitiumSoftware/DnsServer/discussions
+port: 5380
+gallery:
+  - 1.jpg
+  - 2.jpg
+  - 3.jpg
+path: ""
+defaultUsername: ""
+defaultPassword: ""
```

> (NOTE: a full test cannot be done, as the admin password is provided by the $APP_PASSWORD, which cannot be viewed

This is actually where we can set the default credentials so that username and password are shown in the app store when the app is installed, and when a user right clicks on the app icon from the homescreen and selects "Show default credentials".


As an example, say the username is `admin` and we want to show `${APP_PASSWORD}`, then you would do:
```suggestion
defaultUsername: admin
deterministicPassword: true
```